README.md

```diff
@@ -1,8 +1,4 @@
-# Vision Zero Crash Map
-
-This repo houses code for Boston's [vision zero crash map](https://apps.boston.gov/vision-zero/). The data displayed
-on the map can be found on the City's open data portal, under [vision zero crash records](https://data.boston.gov/dataset/vision-zero-crash-records)
-and [vision zero fatality records](https://data.boston.gov/dataset/vision-zero-fatality-records).
+# Abutters App
```

Comment:
I think we can still use some of the readme from vision-zero and just rewrite it, specially the link to staging and prod. Also, some of the description on this PR can be useful.

Reply:
👍added more info to the readme